src/data_type/mod.rs

```diff
@@ -1272,7 +1272,6 @@ impl Variant for Union {
         let fields: BTreeSet<String> = self.fields.iter().map(|(f, _)| f.to_string()).collect();
         let other_fields: BTreeSet<String> =
             other.fields.iter().map(|(f, _)| f.to_string()).collect();
-        assert!(fields.is_subset(&other_fields));
```

Comment:
@ngrislain What is the use of this line ?

Reply:
I think it is reminiscent of a debugging session.
We should indeed not panic in this case, just return false (cf below)

Reply:
Can you remove this?